lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -424,10 +425,18 @@ public function commit($entity = null)
                 }
             }
 
-            $conn->commit();
+            // Commit failed silently
+            if ($conn->commit() === false) {
```

Comment:
In the next major release of DBAL `$conn->commit()` won't return a boolean anymore. Will a silent failure still be a problem there?

Reply:
In 3.x branch of dbal, the commit function is still returning a boolean

Reply:
You're right. I was looking in the master branch. https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Connection.php#L1083